src/math/p5.Vector.js

```diff
@@ -158,7 +207,7 @@ function vector(p5, fn) {
      * </div>
      */
     toString() {
-      return `p5.Vector Object : [${this.x}, ${this.y}, ${this.z}]`;
+      return `p5.Vector Object : [${this.values.join(', ')}]`;
```

Comment:
One of the things I'm considering is to have this be serialized into something more standard/machine readable, so maybe just the JSON part at the end.

Reply:
do you mean something like this?
```
    toString() {
      return `${this.values.join(', ')}]`;
    }
```

Reply:
yeah perhaps, might be worth considering how other libraries serialize their vector objects if they do at all